MAX_ACCELERATION: 3.0
MAX_CURVATURE: 3.0
MAX_D_CURVATURE: 3.0
MAX_YAWRATE: 3.8
